src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/ccd/datamigration/controller/CcdDataMigrationController.java

```diff
@@ -42,14 +43,13 @@ public CallbackResponse migrate(
             AboutToStartOrSubmitCallbackResponse.builder();
 
         // Change the below to point to your new migration service and modify controller tests to match
-        RemoveNottinghamCourtListGaMigration removeNottinghamCourtListGaMigration = new RemoveNottinghamCourtListGaMigration();
-        Map<String, Object> caseData = removeNottinghamCourtListGaMigration.migrate(caseDetails);
+        MigrationHandler migration = new Rpet164Phase1FrcCourtListMigrationImpl();
+        Map<String, Object> caseData = migration.migrate(caseDetails);
```

Comment:
As soon as this is merged, then Dale will not be able to test/run in prod his migration change.
So this either needs to be coordinated with Dales migration change or another endpoint created for this migration.